python/cudf/cudf/tests/test_serialize.py

```diff
@@ -233,3 +233,14 @@ def test_serialize_all_null_string():
 
     recreated = cudf.Series.deserialize(*gd_series.serialize())
     assert_eq(recreated, pd_series)
+
+
+def test_serialize_seriesgroupby_agg():
```

Comment:
Can we just add a basic named series serialize / deserialize roundtrip test? This looks like it's testing the SeriesGroupBy object which I guess tests this transitively?

Reply:
Right - The underlying object of a `SeriesGroupBy` is a named series.  Therefore, the original test was coering both `Series` and `SeriesGroupBy`.  with that said, you are absolutely right that we should make it more obvious what the test is designed to cover.  Therfore, I modified the test name, and added an explicit check for named-series serialization.